srml/contract/src/lib.rs

```diff
@@ -646,49 +705,6 @@ decl_event! {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Contract {
-		/// Number of block delay an extrinsic claim surcharge has.
-		///
-		/// When claim surchage is called by an extrinsic the rent is checked
-		/// for current_block - delay
-		SignedClaimHandicap get(signed_claim_handicap) config(): T::BlockNumber;
-		/// The minimum amount required to generate a tombstone.
-		TombstoneDeposit get(tombstone_deposit) config(): BalanceOf<T>;
-		/// Size of a contract at the time of creation. This is a simple way to ensure
-		/// that empty contracts eventually gets deleted.
-		StorageSizeOffset get(storage_size_offset) config(): u32;
-		/// Price of a byte of storage per one block interval. Should be greater than 0.
-		RentByteFee get(rent_byte_price) config(): BalanceOf<T>;
-		/// The amount of funds a contract should deposit in order to offset
-		/// the cost of one byte.
-		///
-		/// Let's suppose the deposit is 1,000 BU (balance units)/byte and the rent is 1 BU/byte/day,
-		/// then a contract with 1,000,000 BU that uses 1,000 bytes of storage would pay no rent.
-		/// But if the balance reduced to 500,000 BU and the storage stayed the same at 1,000,
-		/// then it would pay 500 BU/day.
-		RentDepositOffset get(rent_deposit_offset) config(): BalanceOf<T>;
-		/// Reward that is received by the party whose touch has led
-		/// to removal of a contract.
-		SurchargeReward get(surcharge_reward) config(): BalanceOf<T>;
-		/// The fee required to make a transfer.
-		TransferFee get(transfer_fee) config(): BalanceOf<T>;
-		/// The fee required to create an account.
-		CreationFee get(creation_fee) config(): BalanceOf<T>;
-		/// The fee to be paid for making a transaction; the base.
-		TransactionBaseFee get(transaction_base_fee) config(): BalanceOf<T>;
-		/// The fee to be paid for making a transaction; the per-byte portion.
-		TransactionByteFee get(transaction_byte_fee) config(): BalanceOf<T>;
-		/// The fee required to create a contract instance.
-		ContractFee get(contract_fee) config(): BalanceOf<T> = 21.into();
```

Comment:
as a note we lost default value and I'm ok with that, but if this is wanted we should be able to modify Get to be able to accept a default from the module.

or just documenting with a good default.

Reply:
We will replace `Get` types with associated consts, as they provide default values and are much better suited for this use case.

Reply:
Doesn't the node/runtime provide good documentation about the defaults?

Reply:
Where? And having a doc comment does prevents you from repeating code that to create this value type.

Reply:
Where the actual values used by the runtime are defined. So what do you want me to do add `Defaults to 21.` to the doc comment?

Reply:
The confusing thing for me is when the documentation suggests one value, but the runtime actually uses a different value. Then I'm not sure why the runtime uses something else. If the default values are all in one place and non contradictory then it's clear what I should be using.

Reply:
The contract fee in the runtime is 1 * CENTS not 21 as an example...